xml/System.Composition.Hosting/ContainerConfiguration.xml

```diff
@@ -345,11 +345,11 @@
         <Parameter Name="conventions" Type="System.Composition.Convention.AttributedModelProvider" />
       </Parameters>
       <Docs>
-        <param name="partTypes">To be added.</param>
-        <param name="conventions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="partTypes">A collection of part types to add.</param>
+        <param name="conventions">The conventions to use for part types.</param>
+        <summary>Adds the specified part types to the container.</summary>
```

Comment:
"Adds the specified part types to the container using the specified conventions."